### PR TITLE
Add links to rapture RPM/Debian lists 

### PR DESCRIPTION
**Pull request by MushuEE**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
Allows release on-call to validate that the packages are visible in public buckets

#### Which issue(s) this PR fixes:
None

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Review by saschagrunert — APPROVED** [review]
/lgtm
Thank you @MushuEE ! :pray:

File: hack/rapture/README.md [hunk 1]
```diff
@@ -104,7 +104,11 @@ The entire build process takes several hours. Once you are ready to begin, the d
 
 ### Validating packages
 
-Now that `rapture` has successfully complete, we need to verify the packages that were just created. This validation can be done on any instance where Kubernetes is not already installed. (Ideally, you would want to spin up a new VM to test.)
+Now that `rapture` has successfully complete, we need to verify the packages that were just published and created correctly.
+
+To check for publish success, see the [Debian package list][deb-package-list] and [RPM package list][rpm-package-list] for the versions that were just uploaded.
```
Comment (amwat) [inline hack/rapture/README.md:109]: probably also include a less manual way?

```
curl https://packages.cloud.google.com/apt/dists/kubernetes-xenial/main/binary-amd64/Packages | grep $version
```

and


```
curl https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64/repodata/primary.xml | grep $version
```

Comment (MushuEE) [reply hack/rapture/README.md:109]: Thanks will do

File: hack/rapture/README.md [hunk 2]
```diff
@@ -103,7 +103,15 @@ The entire build process takes several hours. Once you are ready to begin, the d
 
 ### Validating packages
 
-Now that `rapture` has successfully complete, we need to verify the packages that were just created. This validation can be done on any instance where Kubernetes is not already installed. (Ideally, you would want to spin up a new VM to test.)
+Now that `rapture` has successfully complete, we need to verify the packages that were just published and created correctly.
+
+To check for publish success, see the [Debian package list][deb-package-list] and [RPM package list][rpm-package-list] for the versions that were just uploaded. Or curl on cmdline for Debian and RPM respectively:
+```
+> curl https://packages.cloud.google.com/apt/dists/kubernetes-xenial/main/binary-amd64/Packages | grep $version
```
Comment (amwat) [inline hack/rapture/README.md:110]: drop the `>` signs, makes it harder to just copy paste

File: hack/rapture/README.md [hunk 3]
```diff
@@ -103,7 +103,15 @@ The entire build process takes several hours. Once you are ready to begin, the d
 
 ### Validating packages
 
-Now that `rapture` has successfully complete, we need to verify the packages that were just created. This validation can be done on any instance where Kubernetes is not already installed. (Ideally, you would want to spin up a new VM to test.)
+Now that `rapture` has successfully complete, we need to verify the packages that were just published and created correctly.
+
+To check for publish success, see the [Debian package list][deb-package-list] and [RPM package list][rpm-package-list] for the versions that were just uploaded. Or curl on cmdline for Debian and RPM respectively:
+```
```
Comment (amwat) [inline hack/rapture/README.md:109]: nit:  ` ```shell `

**Review by amwat — COMMENTED** [review]
/lgtm
/hold

**Review by amwat — COMMENTED** [review]
I meant the code block language hint :) but not a big deal.

/lgtm
/hold cancel

**Comment by justaugustus** [comment]
@MushuEE -- this has a merge commit in that needs to be pulled out. Can you also squash the new commits?

File: hack/rapture/README.md (hunk 3 above)
Comment (BenTheElder) [reply hack/rapture/README.md:110]: can we wrap this up into a script command that takes the version instead?
the less manual hackery the better.

Comment (MushuEE) [reply hack/rapture/README.md:110]: Yep I can add a quick one

File: hack/rapture/README.md [hunk 4]
```diff
@@ -105,6 +105,11 @@ The entire build process takes several hours. Once you are ready to begin, the d
 
 Now that `rapture` has successfully complete, we need to verify the packages that were just created. This validation can be done on any instance where Kubernetes is not already installed. (Ideally, you would want to spin up a new VM to test.)
 
+To check for publish success, see the [Debian package list][deb-package-list] and [RPM package list][rpm-package-list] for the versions that were just uploaded. Or curl via shell for Debian and RPM respectively via the [check_rapture script](./check_rapture.sh):
+```shell
+./rapture/check_rapture.sh <version>
```
Comment (amwat) [inline hack/rapture/README.md:110]: nit: This seems to be relative to `hack/` ?

probably either make it relative from the repo root or from this readme.

**Review by amwat — COMMENTED** [review]
/lgtm
/hold

**Review by amwat — COMMENTED** [review]
/lgtm
/hold cancel

**Pull request merged**